__tests__/ExpensiMark-HTML-test.js

```diff
@@ -386,23 +386,15 @@ test('Test quotes markdown replacement with text matching inside and outside cod
 test('Test quotes markdown replacement with text matching inside and outside codefence at the same line', () => {
     const testString = 'The next line should be quoted\n&gt;Hello,I’mtext\nThe next line should not be quoted\n```&gt;Hello,I’mtext```\nsince its inside a codefence';
 
-    const resultString = 'The next line should be quoted<br /><blockquote>Hello,I’mtext</blockquote>The next line should not be quoted<br /><pre>&gt;Hello,I’mtext</pre>since its inside a codefence';
+    const resultString = 'The next line should be quoted<br /><blockquote>Hello,I’mtext</blockquote>The next line should not be quoted <pre>&gt;Hello,I’mtext</pre>since its inside a codefence';
 
     expect(parser.replace(testString)).toBe(resultString);
 });
 
 test('Test quotes markdown replacement with text matching inside and outside codefence at the end of the text', () => {
     const testString = 'The next line should be quoted\n&gt;Hello,I’mtext\nThe next line should not be quoted\n```&gt;Hello,I’mtext```';
 
-    const resultString = 'The next line should be quoted<br /><blockquote>Hello,I’mtext</blockquote>The next line should not be quoted<br /><pre>&gt;Hello,I’mtext</pre>';
-
-    expect(parser.replace(testString)).toBe(resultString);
-});
-
-test('Test quotes markdown replacement with text matching inside and outside codefence with quotes at the end of the text', () => {
```

Comment:
@aneequeahmad why this test was removed? we should only remove the `<br />` from `resultString`, no?

Reply:
> @aneequeahmad could you update those tests? Because those tests are still expecting the `<br/>` tag before `\<pre\>`. Also, could you add tests to cover cases that combine this and the other rule? Something where `<br/>\<pre\>...\</pre\><br/>` will result in `\<pre\>...\</pre\>`

also, could you add a new test case for this case ^?